tests/js/spec/views/settings/project/server-side-sampling/utils/projectStatsToSampleRates.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import {projectStatsToSampleRates} from 'sentry/views/settings/project/server-side-sampling/utils/projectStatsToSampleRates';
+
+describe('projectStatsToSampleRates', () => {
+  it('basic', () => {
```

Comment:
```suggestion
  it('basic', function()  {
```

Reply:
I updated it but I don't think stylistic preferences should be the topic of code review - if we care about having it a specific way, we should create a lint rule.

Reply:
it was just a suggestion 😉